### PR TITLE
new project

### PR DESCRIPTION
**Pull request by Nicolaswg**
my new project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    <meta charset="UTF-8">
+    <title>FCC: Survey</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="item-container">
+      <header class="header">
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+  </header>
```
Comment (Nicolaswg) [inline index.html:13]: I think that for a better HMTL structure, your header tag should be outside your div with the class item-container🏷️

File: index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      </div>
+        <div class="data-container2">
+        <label for="" id="radio-title">Would you recommend freeCodeCamp to a friend</label>
+        <label for="radio1">Definitely</label>
+        <input type="radio" value='Definitely'  id="radio1" name='user-recommend'>
+        <label for="radio2">Maybe</label>
+        <input type="radio" value='Maybe'   id="radio2" name='user-recommend'>
+        <label for="radio3">Not sure</label>
+        <input type="radio" value='Not sure'   id="radio3" name='user-recommend'>
+      </div>
```
Comment (Nicolaswg) [inline index.html:46]: I'll suggest you to use the section tag, this way you can organize better your form input items structure. 🔑

File: index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        <input type="checkbox" value="Videos" id="check7">Videos</label>
+        <label for="check8">
+        <input type="checkbox" value="City Meetups" id="check8">City Meetups</label>
+        <label for="check9">
+        <input type="checkbox" value="Wiki" id="check9">Wiki</label>
+        <label for="check10">
+        <input type="checkbox" value="Forum" id="check10">Forum</label>
+        <label for="check11">
+        <input type="checkbox" value="Additional Courses" id="check11">Additional Courses     </label>
+      </div>
```
Comment (Nicolaswg) [inline index.html:71]: I'll suggest you to use the section tag, this way you can organize better your form input items structure. 🔑

File: index.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+          <p>Any comments or suggestions?</p>
+          <textarea name="" id="text-area" placeholder="Enter your comment here..."></textarea>
+        </div>
+        <div class="button-container">
+          <input type="submit" id="submit" class="button-style">Submit</input>
+        </div>
+      </form>
+    </div>    
+    <script  src="js/index.js"></script>
+  </body>
```
Comment (Nicolaswg) [inline index.html:82]: It is important to know that it isn't necessary to link files that are not in your workflow. I think that you should try to only link the files that you are working on. 🔑

**Review by Nicolaswg — COMMENTED** [review]
## Required Changes ♻️ 

### Professional Repository
For good practices, you should make your repository looks professional and organizes.
 
1. Make your README.md more presentable with a specific description of the entire project.
2. Put screenshots of the last view of your webpage side
3. Make a more descriptive pull request and git commit messages to all the changes you made one by one fill.

Follow the comment below instructions to make a professional and successful project.

*Optional changes are not required but you should read them*

# Linters Errors 🔴 

Submitting projects with existing errors is not a professional practice. I'll recommend you to see if all Linters files are successfully checked for a complete reviewed code.

**Comment by ShoiraTa** [comment]
## Status: Approved :heavy_check_mark:

Hi @Nicolaswg ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. Congratulations!  🎉💯🌟 

## Good Points 👍 
- Being polite with suggestions ✔️ 
- Explaining the reason for requested changes ✔️ 
- Comments are clear ✔️ 
- Using at least 1 emoji to make comments human-friendly ✔️ 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**